src/components/DateTimePicker/Internal/ocpicker.module.scss

```diff
@@ -189,68 +203,180 @@
       opacity: 1;
     }
 
+    &-today:not(&-selected) > &-inner {
+      background: var(--picker-cell-today-background-color);
+      color: var(--picker-cell-today-foreground-color);
+      outline: 1px solid var(--picker-cell-today-background-color);
+
+      &:hover {
+        outline: 1px solid var(--picker-cell-selected-border-color-active);
+      }
+    }
+
     &-in-range {
-      background: $picker-cell-background-color-active;
-      color: $picker-cell-foreground-color-active;
+      background: var(--picker-cell-background-color-hover);
+      box-shadow: 0 -1px 0 0 var(--picker-cell-selected-border-color-active),
+        0 1px 0 0 var(--picker-cell-selected-border-color-active);
+      color: var(--picker-cell-foreground-color-active);
+
+      .picker-cell-inner {
+        outline: none;
+
+        &:hover {
+          outline: none;
+        }
+      }
+
+      &.picker-cell-today {
+        .picker-cell-inner {
+          background: var(--picker-cell-background-color-hover);
+          color: var(--picker-cell-foreground-color-active);
+          outline: none;
+
+          &:hover {
+            outline: none;
+          }
+        }
+      }
+
+      &.picker-cell-range-hover-start,
+      &.picker-cell-range-hover-end {
+        border-bottom-left-radius: 0;
+        border-top-left-radius: 0;
+        outline: 1px solid var(--picker-cell-selected-border-color-active);
+      }
     }
 
     &-range-hover {
-      background: $picker-cell-background-color-active;
-      color: $picker-cell-foreground-color-hover;
+      background: var(--picker-cell-background-color-hover);
+      box-shadow: 0 -1px 0 0 var(--picker-cell-selected-border-color-active),
+        0 1px 0 0 var(--picker-cell-selected-border-color-active);
+      color: var(--picker-cell-foreground-color-hover);
```

Comment:
this is throughout. Going to stop calling them out

Reply:
Done